app/controllers/idv/personal_key_controller.rb

```diff
@@ -60,12 +60,13 @@ def finish_idv_session
     end
 
     def personal_key
+      irs_attempts_api_tracker.idv_personal_key_generated
```

Comment:
Contrasted with where it was located previously, it's not obvious that I'd expect this method to have a side effect of tracking the event, and I'd worry what might happen if it's called multiple times. Is there somewhere else in the action path that we can hook into?

Reply:
Good thoughts - I think the hang-up here is that the event is supposed to track when the personal key is shown to the user for them to download/copy/etc...  The method where it was previously does not seem to be called in a normal user flow - the method should probably be called something like `generate_personal_key_if_not_previously_generated`...  Which then leads to the idea that the event should probably be renamed to something like `idv_personal_key_communicated` since that is what we are actually interested in tracking...

@n1zyy, what do you think?

Reply:
What happens if we move this line up into `finish_idv_session` where we call `personal_key` on line 50? Does that fix this, or is there another hidden caller to this?

Reply:
Good idea - that will help keep this method to be single-minded. 👍